### ISSUE TITLE
skip available update

### ISSUE THREAD

**Issue by top-master** [Microsoft/vscode#57492]
Every Few Minutes we get a new VSCode update available message and it would be nice to have a **skip version** or **Do not show until next update** option available in drop down menu of **Latter** button of the popup message (but it currently does not have any drop-down menu).

although it is very good to have VS Code by it's self up to date but inside of a large project we never do any update except when something is broken, but we do update once the project is done.

**Comment by joaomoreno** [comment]
You can disable updates by changing the setting `update.channel: none`. You can reenable it when you want to update.
